packages/expo-image/src/Image.types.ts

```diff
@@ -70,6 +70,17 @@ export type ImageStyle = RNImageStyle;
  */
 export type ImageContentFit = 'cover' | 'contain' | 'fill' | 'none' | 'scale-down';
 
+/**
+ * Determines downscale dimensions for the image.
```

Comment:
I don't think I understand when I would want to use it from this prop description 😅